about.html

```diff
@@ -12,62 +13,90 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
 
+  <header>
+    <img src="img\lambda-black.png" alt="" width="100px" ;>
+    <nav>
+      <a href="index.html">Home</a>
+      <a href="#">About</a>
+      <a href="#">Products</a>
+      <a href="#">Blog</a>
+      <a href="#">Contact</a>
+    </nav>
+    <img src="img\jumbo-about.png" class="jumbo-image">
+  </header>
 
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
 
-    Learn More
+  <div class="container about-page">
 
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
-      <footer>
-        <nav>
-          <a href="index.html">Home</a>
-          <a href="#">About</a>
-          <a href="#">Products</a>
-          <a href="#">Blog</a>
-          <a href="#">Contact</a>
-        </nav>
-      </footer>
-    </div><!-- container -->        
+  <div class="about">
+    <h2>About</h2>
+    <p> Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to
+      corporate
+      strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic
+      world
+      view of disruptive innovation via workplace diversity and empowerment.</p>
+  </div>
+ 
+
+  <section class="about-boxes">
+
+  <div>
+    <img src="img/about-plan.png" alt="strategy">
```

Comment:
You have the right idea but maybe forgot about it, the alt attribute in each img tag is set to "strategy". For accessibility purposes, it's recommended that the attribute reflect the sections more accurately.